src/components/organisms/UiMultipleChoices/_internal/UiMultipleChoicesItem.vue

```diff
@@ -296,6 +296,11 @@ const optionsToRender = computed(() => props.options.map((option) => ({
   },
 })));
 
+const listItemOptionAttrs = ({
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Instead of disabling the eslint in cases like this, would you add the `ignoreRestSiblings` option to the `no-unused-vars` rule? 
Here's the link to the docs :) 
https://eslint.org/docs/latest/rules/no-unused-vars#ignorerestsiblings

You can do it with this line in the config.
`'@typescript-eslint/no-unused-vars': ['error', { ignoreRestSiblings: true }],`

Reply:
Solved :thank_you: